src/heimdall/models/identity.py

```diff
@@ -0,0 +1,129 @@
+from heimdall.models.model_base import ModelBase
+from heimdall.abstractions.data import (
+    AbstractRepository
+)
+from dateutil.parser import *
+from jsonschema import validate, ValidationError
+
+
+class Identity(ModelBase):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.__load_repository_if_available(kwargs)
+
+    # -------------------------------------------------------------------------
+    # METHOD LOAD REPOSITORY IF AVAILABLE
+    # -------------------------------------------------------------------------
+    def __load_repository_if_available(self, arguments: dict):
+        if 'repository' in arguments and isinstance(arguments['repository'], AbstractRepository):
+            self.__repository = arguments['repository']
+
+    # -------------------------------------------------------------------------
+    # PROPERTY IDENTITY
+    # -------------------------------------------------------------------------
+    @property
+    def __identities(self) -> AbstractRepository:
+        if self.__repository:
+            return self.__repository
+        return None
+
+    # -------------------------------------------------------------------------
+    # PROPERTY ID
+    # -------------------------------------------------------------------------
+    @property
+    def id(self):
+        if self._state and 'identity_id' in self._state:
+            return self._state['identity_id']
+        return None
+
+    # -------------------------------------------------------------------------
+    # PROPERTY BUSINESS ID
+    # -------------------------------------------------------------------------
+    @property
+    def business_id(self):
+        if self._state and 'business_id' in self._state:
+            return self._state['business_id']
+        return None
+
+    # -------------------------------------------------------------------------
+    # PROPERTY LAST_MODIFIED
+    # -------------------------------------------------------------------------
+    @property
+    def last_modified(self):
+        if self._state and 'last_modified' in self._state:
+            return parse(self._state['last_modified'])
+        return None
+
+    # -------------------------------------------------------------------------
+    # PROPERTY DISABLED
+    # -------------------------------------------------------------------------
+    @property
+    def disabled(self):
```

Comment:
This is fine, but would be good to add a return type annotation:

```python
def disabled(self) -> bool:
```
To enforce that returned type should be bool